addons/website/static/src/services/website_custom_menus.js

```diff
@@ -43,6 +43,10 @@ export const websiteCustomMenus = {
                     const displayWebsiteCustomMenu = isWebsiteCustomMenu && website.isRestrictedEditor && this.get(section.xmlid).isDisplayed(env);
                     if (!isWebsiteCustomMenu || displayWebsiteCustomMenu) {
                         let subSections = [];
+                        if (section.xmlid === "website.menu_website_controller_pages_list") {
+                            // TODO: remove the menu item in master.
+                            continue;
+                        }
```

Comment:
Can't we "just" add `active="False"` on the XML menu declaration?
It means that existing DBs not doing a -u would not have the "fix" but:
- It's not really a critical fix, it's just a visual quality of life improvement
- It would avoid hacks in JS which I am never a fan of

What do you think?

Reply:
@rdeodoo Yes, sounds better than excluding the menu item in the custom menus service code for a visual improvement... Let's do it that way! I updated the diff now, thanks for the suggestion :+1: